toolkit/tools/pkg/imagecustomizerlib/extractpartitions.go

```diff
@@ -34,15 +55,36 @@ func extractPartitions(imageLoopDevice string, outDir string, basename string, p
 			case "raw":
 				// Do nothing for "raw" case.
 			case "raw-zst":
-				partitionFilepath, err = compressWithZstd(partitionFilepath)
+				partitionRawFilePath := partitionFilePath
```

Comment:
This is a little too much code for a `case` statement. I recommend putting the code in a helper function.

Reply:
Added a helper function that contains this code, resolving this comment.